assets/src/blocks/Gallery/GalleryCarousel.js

```diff
@@ -1,59 +1,124 @@
-import { Component } from '@wordpress/element';
+import { useState, useEffect, useRef } from '@wordpress/element';
 import _uniqueId from 'lodash/uniqueId';
 
 const { __ } = wp.i18n;
 
-export class GalleryCarousel extends Component {
-  constructor(props) {
-    super(props);
-    this.id = `gallery_${_uniqueId()}`;
+const reflow = element => element.offsetHeight;
```

Comment:
I would document the reflow function in here or in the PR description (maybe just a link to some doc), because it was not obvious when we looked at it :grimacing: